### ISSUE TITLE
No confirmation on exit.

### ISSUE THREAD

**Issue by HighSodium** [felipemanga/ProjectABE#55]
For the love of god, I've lost 2 full-day projects due to a misclick.  Not to mention, none of the buttons are labeled and there is no indication on where files are saved during compile.

2 DAYS.  GONE.

**Comment by felipemanga** [comment]
Can you say a bit more about what happened? Were you using incognito mode and closed the browser? There is no confirmation because, unless you're in incognito mode, everything it's saved in the browser automatically and should be there when you reopen the tab.

**Comment by HighSodium** [comment]
No, this is in the standalone unfortunately.  It auto saves fine in the browser, but not in standalone.  On exit, it wipes all saves and code.

**Comment by felipemanga** [comment]
Ah, standalone. What OS are you on? Is there a directory named "build_" in your temp folder?
I've never seen it erase anything on exit, I'll try to replicate it here. Sorry!

**Comment by HighSodium** [comment]
Window 10.
I'm using the ZIP download not the installer if that makes a difference.

Nothing in the local, low, roaming, or any temp folder that I could find besides the web based files.
I'm able to replicate it with each startup on my end.  Write some code, make a new file, anything, exit program, reload, all is lost.  It works fine in the web based.

Same thing happens if the switch is pressed at the top of the arduboy emulator.
https://imgur.com/a/ZcdM3R5

**Comment by felipemanga** [comment]
Ah, you're on 0.7.9. Have you tried 0.8.10?

**Comment by HighSodium** [comment]
I did today.  While it does save the main file, any secondary files are not saved and erased on exit.  For now I'll stick to the 7.9 or web-based because I can't get 8.10 to build at all.  I've installed the necessary libraries and I can't get it past no 'Arduino IDE found', though I'm probably overlooking something simple.
